### PR TITLE
Add blog section to osv.dev.

### LINKED ISSUE #799: Add a News tab with a blog
Basic thinking: adding a "News" tab to [osv.dev](http://osv.dev/) and making it an OSV-specific blog for more technical posts less suited to the [Google Security Blog](https://security.googleblog.com/). The example that spawned this thought was communicating about #771 once it's completed.

From an internal conversation about this, @oliverchang said: "this has come up a few times, both from our UX designer and from other folks"

### PR DESCRIPTION
**Pull request by oliverchang**
This PR adds https://osv.dev/blog/ and an initial launch post. The blog is written in Markdown and rendered using https://gohugo.io/, and then served using our frontend using our default CSS styling.

Hugo takes care of generating index pages, pagination, rss feeds etc.  

See https://staging-dot-oss-vdb.appspot.com/blog/ for what this looks like.

Fixes #799.

### CODE REVIEW

**Comment by andrewpollock** [comment]
This doesn't seem to produce an RSS feed. I think this should be minimum functionality for a blog.

**Comment by andrewpollock** [comment]
> This doesn't seem to produce an RSS feed. I think this should be minimum functionality for a blog.

(or Atom, I used the term "RSS" generally).

**Comment by oliverchang** [comment]
> > This doesn't seem to produce an RSS feed. I think this should be minimum functionality for a blog.
> 
> (or Atom, I used the term "RSS" generally).

There is an atom feed auto-generated, but I wasn't exposing it. I've fixed it and now it's available at https://staging-dot-oss-vdb.appspot.com/blog/index.xml. I can add a link to this somewhere in a subsequent PR.

File: gcp/appengine/frontend_handlers.py [hunk 1]
```diff
@@ -88,6 +103,29 @@ def index():
       'home.html', ecosystem_counts=osv_get_ecosystem_counts_cached())
 
 
+@blueprint.route('/blog/', strict_slashes=False)
+def blog():
+  return render_template('blog.html', index=_load_blog_content('index.html'))
+
+
+@blueprint.route('/blog/index.xml')
```
Comment (andrewpollock) [inline gcp/appengine/frontend_handlers.py:111]: How does a post wind up in the feed?

Comment (oliverchang) [reply gcp/appengine/frontend_handlers.py:111]: It's all automatic as part of hugo.

Comment (andrewpollock) [reply gcp/appengine/frontend_handlers.py:111]: Very cool!

**Review by andrewpollock — APPROVED** [review]
I'm assuming these CodeQL findings are nothing to be concerned about?

**Comment by oliverchang** [comment]
> I'm assuming these CodeQL findings are nothing to be concerned about?

Right, these values are already validated earlier before this is reached.

**Pull request merged**